pagopa/pn-frontend
# PN-10943-fe-api-annullamento-notifica-pa

## Short Description
Migrate the old cancel notification API to the BFF's API.

## List of Changes Proposed in This Pull Request
- Generated models from the BFF OpenAPI.
- Updated `cancelNotification()` with the BFF's API.
- Updated related tests.

## How to Test
Start PA and check if everything works as intended (i.e: cancel a notification).
